services/app-web/src/pages/StateSubmission/SubmissionType/SubmissionType.tsx

```diff
@@ -155,7 +155,13 @@ export const SubmissionType = ({
 
     const handleFormSubmit = async (
         values: SubmissionTypeFormValues,
-        formikHelpers: FormikHelpers<SubmissionTypeFormValues>
+        formikHelpers: Pick<
+            FormikHelpers<SubmissionTypeFormValues>,
+            'setSubmitting'
+        >,
```

Comment:
Changed the `formikHelpers` type to pick out `setSubmitting` used in the function. I couldn't figure out a way, if there is one, to pass the helpers from `formik` to match `FromikHelpers` type.

Reply:
I don't get what FormikHelpers are, what are we using them for elsewhere?

Reply:
They are a way to access formik state. Read more here https://formik.org/docs/api/withFormik#the-formikbag  - whenever we submit a form with formik theres a bunch of state we have access to.

 In terms of how we are using it looks like in Contacts an Submission Type forms only -- which suggests it might be able to be refactored out if we unify the forms.

Reply:
@JasonLin0991 FYI the type of `setSubmitting` is literally just `setSubmitting: (isSubmitting: boolean) => void`.  The way you are doing things here also works, that is the proper way to pick out a type from a set of typed properties. I think what you are doing works well.

> I couldn't figure out a way, if there is one, to pass the helpers from formik to match FromikHelpers type.

I _believe_ the only way you could get that type is directly from the parameter within the` <Formik onSubmit />`
![Screen Shot 2022-04-28 at 8 44 41 AM](https://user-images.githubusercontent.com/10750442/165766627-f4b9b1d7-f97c-4fa5-9722-a82acb9f9d79.png)


Hope that answers your type question. If not, let's look together when we pair.